### DISCUSSION TITLE
SQLite Studio 3.3.3 shows incomplete table

### DISCUSSION THREAD

**Discussion by cqcq42** [pawelsalawa/sqlitestudio#4870] [Q&A]
### Details
SQLite Studio is not reading all the data in my database (only 1000 of 3431 records)
Another program can query this database get all rows.

### Steps to reproduce
Open the attached TEST.txt file as a database. (Had to change extension to fit your file requirements.)

### Operating system
Windows 10 22H2 build 19045.3324

### SQLiteStudio version
3.3.3

[TEST.txt](https://github.com/pawelsalawa/sqlitestudio/files/12577097/TEST.txt)

**Comment by pawelsalawa** [comment]
Just making sure - are you aware of moving between result pages (and having 1000 result rows per page)?
![image](https://github.com/pawelsalawa/sqlitestudio/assets/481846/cf28ec45-8064-4d9f-acf3-8ad493bf58c2)

**Comment by cqcq42** [comment]
OK. Did not know that. Thanks for the tip.GregThere is no hope!(But I may be wrong.)On Sep 11, 2023, at 11:41 AM, Paweł Salawa ***@***.***> wrote:﻿
Just making sure - are you aware of moving between result pages (and having 1000 result rows per page)?
